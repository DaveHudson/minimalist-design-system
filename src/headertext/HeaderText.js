import React, { Component } from 'react';
import PropTypes from 'prop-types';

class HeaderTextElement extends Component {
  render() {
    return (
      <p>Hello there!</p>
    );
  }
}

export default HeaderTextElement;
