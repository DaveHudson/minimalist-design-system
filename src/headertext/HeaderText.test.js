import React from 'react';
import renderer from 'react-test-renderer';

import HeaderText from './HeaderText';

describe('HEADER TEXT', () => {
  it('Renders with DefaultProps', () => {
    const component = renderer
      .create(<HeaderText />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
