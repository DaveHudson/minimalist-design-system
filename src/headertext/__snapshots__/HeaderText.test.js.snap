// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HEADER TEXT Renders with DefaultProps 1`] = `
<p>
  Hello there!
</p>
`;
